contracts/test/integration/utils/TimelockSimulation.sol

```diff
@@ -0,0 +1,96 @@
+pragma solidity =0.8.13;
+
+import {TimelockController} from "@openzeppelin/contracts/governance/TimelockController.sol";
+import {DSTest} from "./../../unit/utils/DSTest.sol";
+import {Vm} from "./../../unit/utils/Vm.sol";
+
+import "hardhat/console.sol";
+
+contract TimelockSimulation is DSTest {
+    /// an array of actions makes up a proposal
+    struct action {
+        address target;
+        uint256 value;
+        bytes arguments;
```

Comment:
Minor, but I'd probably rename `arguments` to `payload` or `data`. This 'arguments' field is the abi encoded data that hits the function selector `abi.encodeWithSelector('function add(uint256,uint256)', 5, 10)` - not just some args passed in